src/custom/state/claim/hooks/index.ts

```diff
@@ -23,27 +23,29 @@ export { useUserClaimData } from '@src/state/claim/hooks'
 const CLAIMS_REPO_BRANCH = 'main'
 export const CLAIMS_REPO = `https://raw.githubusercontent.com/gnosis/cow-merkle-drop/${CLAIMS_REPO_BRANCH}/`
 
-export const enum ClaimType {
-  Airdrop = 'Airdrop', // free, no vesting, can be available on both mainnet and gchain
-  GnoOption = 'GnoOption', // paid, with vesting, must use GNO, can be available on both mainnet and gchain
-  UserOption = 'UserOption', // paid, with vesting, must use Native currency, can be available on both mainnet and gchain
-  Investor = 'Investor', // paid, with vesting, must use USDC, only on mainnet
-  Team = 'Team', // free, with vesting, only on mainnet
-  Advisor = 'Advisor', // free, with vesting, only on mainnet
+// TODO: these values came from the test contract, might be different on real deployment
+export const WETH_PRICE = '37500000000000' // '0.0000375' WETH (18 decimals) per vCOW, in wei
+export const GNO_PRICE = '375000000000000' // '0.000375' GNO (18 decimals) per vCOW, in atoms
+export const USDC_PRICE = '150000' // '0.15' USDC (6 decimals) per vCOW, in atoms
+
+// Constants regarding investment time windows
+const ONE_WEEK = 7 * 24 * 60 * 60 * 1000 // in ms
+const TWO_WEEKS = 2 * ONE_WEEK // in ms
+const SIX_WEEKS = 6 * ONE_WEEK // in ms
+
+const TEN_TO_EIGHTEENTH_POWER = JSBI.exponentiate(JSBI.BigInt('10'), JSBI.BigInt('18'))
```

Comment:
we have `formatUnits` and `parseUnits` from `ethers` that i think you want this for

Reply:
`parseUnits` being the relevant one